mongodb/build.gradle.kts

```diff
@@ -0,0 +1,16 @@
+plugins {
+  id("org.hypertrace.docker-plugin") version "0.6.1"
+  id("org.hypertrace.docker-publish-plugin") version "0.6.1"
+}
+
+// MongoDB was last available in the 3.9 release
+// https://pkgs.alpinelinux.org/packages?name=mongodb&branch=v3.9
+var mongodbVersion = "4.0.5-r0"
```

Comment:
BTW, we're using `4.4.0` mongo currently in Hypertrace.

Reply:
FYI undid this parameterization as it is unlike the other images.

Anyway the version we used before was this (in docker-compose) 4.0.5 and the main mongo image is massive. I haven't researched an alternative actually slim image for mongo because I keep hearing mongo isn't a long term thing.

Reply:
There is some folklore about Mongo restricting license which short circuited non-official images. https://github.com/DannyBen/docker-alpine-mongo/issues/1#issuecomment-581010564

4.0.5 coincides roughly the same time with them switching to a less friendly license https://www.mongodb.com/licensing/server-side-public-license

I wouldn't recommend doing more on Mongo unless it is decided a long term storage option. If decided a permanent option, try to get mongo to make a less massive image is probably a good bet.